src/libs/form/components/NumberSlider.tsx

```diff
@@ -32,21 +42,32 @@ export function NumberSlider({
   sliderProps,
   numberProps,
   reverse,
+  presets,
+  label,
   ...inputWrapperProps
 }: NumberSliderProps) {
   const { classes, cx } = useStyles();
   const numberRef = useRef<HTMLInputElement>(null);
-  const [focused, setFocused] = useState(false);
-  const [_value, setValue] = useState(value);
-  const [changeEndValue, setChangeEndValue] = useState<number>();
-  const [computedWidth, setComputedWidth] = useState<string>();
+  const [state, setState] = useState<State>({
+    focused: false,
+    _value: value,
```

Comment:
There's no advange at using `_` now that you have it all under a state 🤷 Can be`value` imho